core/src/main/java/hudson/util/FormApply.java

```diff
@@ -87,4 +93,31 @@ public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object nod
             }
         };
     }
+
+    /**
+     * Generates the response for the asynchronous background form submission (AKA the Apply button),
+     * that will show a notification of certain type and with provided message.
+     * @since TODO
+     */
+    public static HttpResponseException showNotification(final String message, final NotificationType notificationType) {
```

Comment:
@yaroslavafenkin Make it clear through naming and/or Javadoc what the HTML content of `message` is. The call looks like it would support HTML, but needs to be documented.

Reply:
What do you mean by the HTML content of `message`? I'm not catching it.

Reply:
Is `message` plain text/escaped, or does it support HTML? There's no Javadoc for the argument, so this is undetermined.